server/models.py

```diff
@@ -776,13 +776,20 @@ class Backup(Model):
     id = db.Column(db.Integer, primary_key=True)
 
     submitter_id = db.Column(db.ForeignKey("user.id"), nullable=False)
+    # NULL if same as submitter
+    creator_id = db.Column(db.ForeignKey("user.id"), nullable=True)
     assignment_id = db.Column(db.ForeignKey("assignment.id"), nullable=False)
     submit = db.Column(db.Boolean(), nullable=False, default=False)
     flagged = db.Column(db.Boolean(), nullable=False, default=False)
 
     extension = db.Column(db.Boolean(), default=False)
```

Comment:
Should we remove this property? Once we get full extension support - maybe it'll be useful but for now - it seems like it won't be